kotlin/src/main/kotlin/com/linecorp/armeria/internal/common/kotlin/ArmeriaRequestCoroutineContext.kt

```diff
@@ -25,6 +25,7 @@ import kotlin.coroutines.CoroutineContext
 /**
  * Propagates [ServiceRequestContext] over coroutines.
  */
+@Deprecated("Use RequestContext.asCoroutineContext() instead.", ReplaceWith("RequestContext.asCoroutineContext()"))
```

Comment:
Question) Since this is in the internal package, is it difficult to just remove this file?

Reply:
This class is in an internal package but it's used by users. So I think it's safe to deprecate it first. 😉